apps/registration/src/components/Button.tsx

```diff
@@ -0,0 +1,31 @@
+const Button = ({
+  onClick,
+  label,
```

Comment:
change label => name, so it follows other component

Reply:
Roger that krub